.gitignore

```diff
@@ -0,0 +1,2 @@
+target
+.idea
```

Comment:
The [archetype](https://github.com/jenkinsci/archetypes/blob/2d125f8a7aa41ea38f75704d463f74b887cc92a5/common-files/gitignore) has a few more entries:

```suggestion
target

# mvn hpi:run
work

# IntelliJ IDEA project files
*.iml
*.iws
*.ipr
.idea

# Eclipse project files
.settings
.classpath
.project
```